src/shared.rs

```diff
@@ -66,9 +65,15 @@ pub type HResult<T> = Result<T, HyprError>;
 /// and has methods to reveal the address in different data formats
 #[derive(Debug, Deserialize, Serialize, Clone, Eq, PartialEq, Hash, Ord, PartialOrd)]
 pub struct Address(String);
+impl Address {
+    #[inline(always)]
+    pub(crate) fn fmt_new(address: &str) -> Self {
+        // this way is faster than std::fmt
+        Self("0x".to_owned() + address)
+    }
+}
```

Comment:
why is it `fmt_new` and not `new`

Reply:
I wanted to make the distinction. All this is doing is formatting a string slice into a String with the proper prefix. I could change it to new, and I will if you want.

Reply:
actually now that I am looking at the code again, I remember -- new() was already defined.

Reply:
hyprland always has a prefix tho

Reply:
After thinking some more, keeping it as `fmt_new` is fine